ghc-typelits-presburger 0.3.0.0

```
--  While building package ghc-typelits-presburger-0.3.0.0 using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.2.0.0_
ghc-8.10.1 --builddir=.stack-work/dist/x86_64-linux/Cabal-3.2.0.0 build lib:ghc-typeli
ts-presburger --ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/ghc-type
lits-presburger-0.3.0.0.log

    Configuring ghc-typelits-presburger-0.3.0.0...
    Preprocessing library for ghc-typelits-presburger-0.3.0.0..
    Building library for ghc-typelits-presburger-0.3.0.0..
    [2 of 5] Compiling GHC.TypeLits.Presburger.Compat
    
    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:10:63: error:
        Module ‘GhcPlugins’ does not export ‘EqRel(..)’
       |
    10 | import GhcPlugins          as GHC.TypeLits.Presburger.Compat (EqRel (..), Pre
dTree (..))
       |                                                               ^^^^^^^^^^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:10:75: error:
        Module ‘GhcPlugins’ does not export ‘PredTree(..)’
       |
    10 | import GhcPlugins          as GHC.TypeLits.Presburger.Compat (EqRel (..), Pre
dTree (..))
       |                                                                           ^^^
^^^^^^^^^^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:11:63: error:
        Module ‘GhcPlugins’ does not export ‘classifyPredType’
       |
    11 | import GhcPlugins          as GHC.TypeLits.Presburger.Compat (classifyPredTyp
e, isEqPred,
       |                                                               ^^^^^^^^^^^^^^^
^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:11:81: error:
        Module ‘GhcPlugins’ does not export ‘isEqPred’
       |
    11 | import GhcPlugins          as GHC.TypeLits.Presburger.Compat (classifyPredTyp
e, isEqPred,
       |
   ^^^^^^^^

   /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:24:63: error:
        Module ‘TcRnMonad’ does not export ‘Ct’
       |
    24 | import TcRnMonad           as GHC.TypeLits.Presburger.Compat (Ct, TcPluginRes
ult (..), isWanted)
       |                                                               ^^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:24:88: error:
        Module ‘TcRnMonad’ does not export ‘isWanted’
       |
    24 | import TcRnMonad           as GHC.TypeLits.Presburger.Compat (Ct, TcPluginRes
ult (..), isWanted)
       |
          ^^^^^^^^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:25:78: error:
        Module ‘TcRnTypes’ does not export ‘ctEvPred’
       |
    25 | import TcRnTypes           as GHC.TypeLits.Presburger.Compat (TcPlugin (..),
ctEvPred, ctEvidence)
       |
^^^^^^^^

    /var/stackage/work/unpack-dir/unpacked/ghc-typelits-presburger-0.3.0.0-20fecd68ff0
be5884bf8bf5b8cfd52eb6be9d6f197c5735294e6079803a4263e/src/GHC/TypeLits/Presburger/Comp
at.hs:25:88: error:
        Module ‘TcRnTypes’ does not export ‘ctEvidence’
       |
    25 | import TcRnTypes           as GHC.TypeLits.Presburger.Compat (TcPlugin (..),
ctEvPred, ctEvidence)
       |
          ^^^^^^^^^^			        
```

--- a/build-constraints.yaml
+++ b/build-constraints.yaml
@@ -4250,10 +4250,10 @@ packages:
 
     "Hiromi Ishii <konn.jinro@gmail.com> @konn":
         - equational-reasoning
-        - ghc-typelits-presburger < 0 # ghc 8.10.1 https://github.com/commercialhaskell/stackage/issues/5438
-        - singletons-presburger < 0 # via ghc-typelits-presburger
-        - type-natural < 0 # ghc 8.10
-        - sized < 0 # ghc 8.10
+        - ghc-typelits-presburger
+        - singletons-presburger
+        - type-natural
+        - sized
 
     "Frank Doepper <stackage@woffs.de> @woffs":
         - amqp-utils

